tests/test_workspaces/test_apis/test_export_settings/test_triggers.py

```
@@ -96,3 +98,131 @@ def test_post_save_workspace_general_settings_export_trigger_3(mocker, db):
 
     assert after_errors_count == 0
     assert after_delete_count == before_delete_count - expense_grp_personal_count
+
+
+def test_run_pre_save_expense_group_setting_triggers_no_existing_settings(db, mocker):
```

Ruff (0.14.0)

Unused function argument: `db`

(ARG001)